django-verdant/rcasite/settings/docker.py

```diff
@@ -0,0 +1,270 @@
+import os
+import dj_database_url
+import raven
+
+from .base import *  # noqa
+
+# Do not set SECRET_KEY, Postgres or LDAP password or any other sensitive data here.
+# Instead, use environment variables or create a local.py file on the server.
+
+# Disable debug mode
+DEBUG = True
+
+
+# Use cached template loader
+TEMPLATE_LOADERS = (
+    ('django.template.loaders.cached.Loader', (
+        'django.template.loaders.filesystem.Loader',
+        'django.template.loaders.app_directories.Loader',
+    )),
+)
+
+
+# Send emails from publications@rca.ac.uk via gmail.com
+DEFAULT_FROM_EMAIL = 'publications@rca.ac.uk'
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = DEFAULT_FROM_EMAIL
+EMAIL_HOST_PASSWORD = ''
+
+
+# LDAP authentication
+from rca_ldap.settings import *
+
+AUTH_LDAP_BIND_DN = ''
+AUTH_LDAP_BIND_PASSWORD = ''
+AUTH_LDAP_SERVER_URI = ''
+
+AUTHENTICATION_BACKENDS = (
+    'django_auth_ldap.backend.LDAPBackend',
+    'django.contrib.auth.backends.ModelBackend',
+)
+
+
+# Google Analytics
+GOOGLE_ANALYTICS_ACCOUNT = 'UA-3809199-5'
+
+SILVERPOP_ID = 'd871c05-15671ce6a9e-c6f842ded9e6d11c5ffebd715e129037'
+SILVERPOP_BRANDEDDOMAINS = 'www.pages05.net,rcawebsite,studentenquiry97.mkt4116.com'
+
+
+# Compress static files offline and minify CSS
+# http://django-compressor.readthedocs.org/en/latest/settings/#django.conf.settings.COMPRESS_OFFLINE
+COMPRESS_OFFLINE = True
```

Comment:
I think this _is_ what we wanted, compressing offline

Reply:
I've used a `docker.py` file for the settings at the moment to use with docker-compose

Reply:
Yep, but it is already in base.py (https://github.com/torchbox/verdant-rca/blob/master/django-verdant/rcasite/settings/base.py#L217) so shouldn't need to repeat it here.